VisaCheckout/src/test/java/com/braintreepayments/api/VisaCheckoutClientUnitTest.kt

```diff
@@ -0,0 +1,172 @@
+package com.braintreepayments.api
+
+import com.braintreepayments.api.Configuration.Companion.fromJson
+import com.braintreepayments.api.TestConfigurationBuilder.TestVisaCheckoutConfigurationBuilder
+import com.visa.checkout.Profile.CardBrand
+import com.visa.checkout.VisaPaymentSummary
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.verify
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertNotNull
+import org.json.JSONException
+import org.json.JSONObject
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.util.Arrays
+import java.util.concurrent.CountDownLatch
+
+@RunWith(RobolectricTestRunner::class)
+class VisaCheckoutClientUnitTest {
+
+    private lateinit var configurationWithVisaCheckout: Configuration
+    private lateinit var visaPaymentSummary: VisaPaymentSummary
+
+    @Before
+    @Throws(Exception::class)
+    fun setup() {
+        configurationWithVisaCheckout = fromJson(Fixtures.CONFIGURATION_WITH_VISA_CHECKOUT)
+        visaPaymentSummary = mockk(relaxed = true)
+
+        every { visaPaymentSummary.callId } returns "stubbedCallId"
+        every { visaPaymentSummary.encKey } returns "stubbedEncKey"
+        every { visaPaymentSummary.encPaymentData } returns "stubbedEncPaymentData"
+
+//        every { VisaPaymentSummary::class.java}.withAnyArguments() returns visaPaymentSummary)
```

Comment:
Do we need this?

Reply:
This doesn't appear to be needed. The original test [stubs the constructor with powermock](https://github.com/braintree/braintree_android/blob/e139d80dbe193eb568fb08dbd365d049ccc201e4/VisaCheckout/src/test/java/com/braintreepayments/api/VisaCheckoutClientUnitTest.java#L57), but I checked and we don't invoke the constructor internally, so the stub has no effect. I'm going to remove it.